lib/eslint-path.js

```diff
@@ -9,6 +9,10 @@ exports.resolve = function (cwd, eslint_path) {
   try {
     return resolver.resolve(eslint_path, { paths: [cwd] });
   } catch (e) {
+    if (process.env.ESLINT_D_LOCAL_ONLY) {
+      return undefined;
```

Comment:
This case needs to be handled here:

https://github.com/mantoni/eslint_d.js/blob/bcb6b1f546fa6208835b521974adc08957640b44/lib/linter.js#L87-L96

Reply:
Should it just just return early before setting the cache?

Reply:
Hmm, I guess that makes sense. Otherwise each directory where `eslint_d` is invoked without doing anything would be added to the cache, potentially evicting other instances from the LRU cache.